references/detection/train.py

```diff
@@ -47,8 +47,15 @@ def get_dataset(name, image_set, transform, data_path):
     return ds, num_classes
 
 
-def get_transform(train):
-    return presets.DetectionPresetTrain() if train else presets.DetectionPresetEval()
+def get_transform(train, args):
```

Comment:
No worries, I will pass explicitly the parameter that I need then. 

I'm currently testing the SSD Augmentation transforms to FasterRCNN. If it proves beneficial, I will instead create a new arg that turns on the augmentation so that it can be used by other models.

Reply:
Introduced a `--data-augmentation` parameter on the script that defaults to "hflip" (only horizontal flips). If someone passes an "ssd" argument it will do the new augmentation. This will allow us to implement more augmentation strategies on the future and make them selectable from the scripts.